ruby-challenges.rb

```diff
@@ -0,0 +1,77 @@
+# ASSESSMENT 4: Ruby Coding Practical Questions
+# MINASWAN ✌️
+
+# --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
+# need to take the number as an argument and run it thru a conditional that checks if it is even or odd and return the expected statement
+# I used the modulo by 2 to see if it is even or odd
+def is_even num
+  if num % 2 == 0
+    p "#{num} is even"
+  else 
+    p "#{num} is odd"
+  end
+end
+num1 = 7
+p is_even num1
+# if num1 % 2 != 0
+#   p "#{num1} is odd"
+# end
+# it works
+# Expected output: '7 is odd'
+# same thing here
+num2 = 42
+p is_even num2
+# if num2 % 2 == 0 
+#   p "#{num2} is even"
+# end
+# it works
+# Expected output: '42 is even'
+# same thing here
+num3 = 221
+p is_even num3
+# if num3 % 2 != 0
+#   p "#{num3} is odd"
+# end
+# Expected output: '221 is odd'
+# I'm not sure if there was anything else that was required here, so I'm just going to make it on conditional method and just plug it in to each num.
+# Also not sure why I'm getting double prints.. but I'll rather be twice right then not at all?
```

Comment:
Logic looks great, the double prints is because you are printing both inside the method and printing the invocation.  Best practice would be to only print when invoking the method.